zipkin-query/src/test/scala/com/twitter/zipkin/query/ZipkinQueryServerFeatureTest.scala

```diff
@@ -876,4 +876,11 @@ class ZipkinQueryServerFeatureTest extends FeatureTest with MockitoSugar with Be
       andExpect = Ok,
       withJsonBody = "[ ]")
   }
+
+  "health checks OK" in {
+    server.httpGet(
+      path = "/health",
+      andExpect = Ok,
+      withBody = "OK\n")
```

Comment:
I usually see only `OK` but does nor really matter, I do not think any tool would check anything other than the 200 response